Classes/common/HTTP/CDTHTTPInterceptorContext.m

```diff
@@ -16,6 +16,12 @@
 
 #import "CDTHTTPInterceptorContext.h"
 
+@interface CDTHTTPInterceptorContext ()
+
+@property (nullable, readwrite, nonatomic, strong) NSURLResponse *response;
```

Comment:
Does this property need to be re-declared here for local read/write access? Or can you just access `_response` directly?